File: app/src/main/java/com/ashraf/whatsappvpn/AppConfig.java
package com.ashraf.whatsappvpn;

import android.app.Application;
import com.tencent.mmkv.MMKV;

public class AppConfig extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        // 🎯 [CRITICAL FIX] वापस आई mmkv.aar लाइब्रेरी को यहाँ इनिशियलाइज कर दिया है ताकि ऐप क्रैश न हो
        MMKV.initialize(this);
    }
}

File: app/src/main/java/com/ashraf/whatsappvpn/AppConfig.kt
package com.ashraf.whatsappvpn

import android.app.Application

class AppConfig : Application() {
    override fun onCreate() {
        super.onCreate()
        // पुरानी लाइब्रेरीज (MMKV और Core) वाला कोड यहाँ से हटा दिया गया है
        // क्योंकि उनकी अब कोई ज़रूरत नहीं है।
    }
}
